README.md

```diff
@@ -59,6 +59,10 @@ module "apiary" {
     }
   ]
   apiary_customer_accounts = ["aws_account_no_1", "aws_account_no_2"]
+  apiary_customer_condition = <<EOF
+    "ForAnyValue:StringEquals": {"s3:ExistingObjectTag/security": [ "public"] } ,
+    "StringLike": {"s3:ExistingObjectTag/type": "image*" }
```

Comment:
I have it as 'OBJECT_TYPE' in my code, but I'll change it to 'type'

Reply:
variable is generic, we can use OBJECT_TYPE internally, above is just a example to show usage

Reply:
Oh this is in the README, didn't notice...